static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -68,16 +68,15 @@ define([
         }
     };
 
-    policies.membershipMessages = function () {
-        if (!detect.adblockInUseSync() &&
-            detect.getBreakpoint() !== 'mobile' &&
-            config.page.contentType === 'Article' &&
-            !userFeatures.isPayingMember()
-        ) {
-            return {
-                membershipMessages : true
-            };
-        }
+    policies.checkWeCanShowMembershipMessages = function () {
```

Comment:
Still confused how this is a function called `checkWeCanShowMembershipMessages` that returns an object with a property of the same name, `checkWeCanShowMembershipMessages`.

I generally think that bad abstractions are worse than duplication of code. Would it be better just to have a `checkWeCanShowMembershipMessages` function that returns a `Promise[Boolean]` and worry about abstracting it when/if we need to?